### PR TITLE
fix: new checkbox component

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/5238

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
In a lot of places where there was a checkbox, now there is a switched

**Review by sachinchauhan2889 — COMMENTED** [review]
checkbox are not shown on admin events page.

admin checkbox

**Comment by maze-runnar** [comment]
@sachinchauhan2889 I think this issue is not due to this PR. I encountered the issue on eventyay site too.
Screenshot from 2021-01-04 22-01-27

**Comment by iamareebjamal** [comment]
@maze-runnar Please open an issue for it and fix it

**Comment by iamareebjamal** [comment]
Not working https://open-event-frontend-git-fork-maze-runnar-checkbox-doc.eventyay.now.sh/admin/permissions/event-roles

**Comment by mariobehling** [comment]
Working here now https://open-event-frontend-git-fork-maze-runnar-checkbox-doc-eventyay.vercel.app/admin/permissions/event-roles

But: I am not sure if I checked all places that are affected by this change. Looks good for me generally. Thanks

**Comment by mariobehling** [comment]
@maze-runnar Please resolve conflicts.

**Comment by MeghalBisht** [comment]
Not on frontpage still uses uiCheckbox is this intended @maze-runnar ?
Screenshot from 2021-05-18 21-05-15

**Comment by MeghalBisht** [comment]
Side bar was updated after this PR. Please change these 2 filters
Screenshot from 2021-05-18 21-57-01

File: app/templates/components/forms/admin/settings/images-form.hbs [hunk 1]
```diff
@@ -16,7 +16,7 @@
     </div>
   </div>
   <div class="field">
-    <UiCheckbox @label={{t "Standard aspect ratio is 13:5. Tick to maintain aspect ratio."}} @class="checkbox" @name="large_ratio" @checked={{this.image.eventImageSize.fullAspect}} @onChange={{action (mut this.image.eventImageSize.fullAspect)}} />
+    <LegacyCheckbox @label={{t "Standard aspect ratio is 13:5. Tick to maintain aspect ratio."}} @class="checkbox" @name="large_ratio" @checked={{this.image.eventImageSize.fullAspect}} @onChange={{action (mut this.image.eventImageSize.fullAspect)}} />
```
Comment (MeghalBisht) [inline app/templates/components/forms/admin/settings/images-form.hbs:19]: Cannot untick here.

File: app/templates/components/forms/admin/settings/images-form.hbs [hunk 2]
```diff
@@ -39,7 +39,7 @@
     </div>
   </div>
   <div class="field">
-    <UiCheckbox @label={{t "Standard aspect ratio is 5:2. Tick to maintain aspect ratio."}} @class="checkbox" @name="thumb_ratio" @checked={{this.image.eventImageSize.thumbnailAspect}} @onChange={{action (mut this.image.eventImageSize.thumbnailAspect)}} />
+    <LegacyCheckbox @label={{t "Standard aspect ratio is 5:2. Tick to maintain aspect ratio."}} @class="checkbox" @name="thumb_ratio" @checked={{this.image.eventImageSize.thumbnailAspect}} @onChange={{action (mut this.image.eventImageSize.thumbnailAspect)}} />
```
Comment (MeghalBisht) [inline app/templates/components/forms/admin/settings/images-form.hbs:42]: Cannot untick here.

File: app/templates/components/forms/admin/settings/images-form.hbs [hunk 3]
```diff
@@ -64,7 +64,7 @@
     </div>
   </div>
   <div class="field">
-    <UiCheckbox @label={{t "Standard aspect ratio is 5:2. Tick to maintain aspect ratio."}} @class="checkbox" @name="icon_ratio" @checked={{this.image.eventImageSize.iconAspect}} @onChange={{action (mut this.image.eventImageSize.iconAspect)}} />
+    <LegacyCheckbox @label={{t "Standard aspect ratio is 5:2. Tick to maintain aspect ratio."}} @class="checkbox" @name="icon_ratio" @checked={{this.image.eventImageSize.iconAspect}} @onChange={{action (mut this.image.eventImageSize.iconAspect)}} />
```
Comment (MeghalBisht) [inline app/templates/components/forms/admin/settings/images-form.hbs:67]: Cannot untick here.

**Comment by MeghalBisht** [comment]
Checkbox is needed here
Screenshot from 2021-05-18 22-07-25

File: app/templates/admin/permissions/system-roles.hbs [hunk 4]
```diff
@@ -93,9 +93,9 @@
                 {{userPermission.description}}
               </div>
             </td>
-            <td><UiCheckbox @label={{t "Unverified User"}} @checked={{userPermission.unverifiedUser}} @onChange={{action (mut userPermission.unverifiedUser)}} /></td>
+            <td><LegacyCheckbox @class="ui checkbox" @label={{t "Unverified User"}} @checked={{userPermission.unverifiedUser}} @onChange={{action (mut userPermission.unverifiedUser)}} /></td>
             <td>
-              <UiCheckbox @label={{t "Anonymous User"}} @disabled={{true}} />
```
Comment (MeghalBisht) [inline app/templates/admin/permissions/system-roles.hbs:98]: cant untick

**Comment by maze-runnar** [comment]
> `Not on frontpage` still uses uiCheckbox is this intended @maze-runnar ?
> Screenshot from 2021-05-18 21-05-15

No, need to be updated.

File: app/templates/admin/permissions/system-roles.hbs (hunk 4 above)
Comment (maze-runnar) [reply app/templates/admin/permissions/system-roles.hbs:98]: > Side bar was updated after this PR. Please change these 2 filters
> Screenshot from 2021-05-18 21-57-01

ok

**Comment by maze-runnar** [comment]
will check it later.

**Pull request closed without merging**